src/js/stories/CampaignCard.stories.tsx

```diff
@@ -5,6 +5,8 @@ import { Container, Row, Col, CampaignCard } from '../';
 
 import '../../scss/ray.scss';
 
+// TODO add types in fonction of the tag passed ? How to do that ?????
```

Comment:
Can you explain this more?

Reply:
Oh yeah it was a reminder that I put early on because of the issue we discussed. If the tag is `a` we would need to accept href as a props. but for now, the issue is fixed with `[key: string]: any;` 
I'll remove it